config/initializers/statsd.rb

```diff
@@ -25,6 +25,20 @@ class << Samson
   Samson.statsd.histogram "execute_shell.time", event.duration, tags: tags
 end
 
+[
+  "execute.command_executor.samson",
+  "execute.terminal_executor.samson",
+  "request.rest_client.samson",
+  "request.vault.samson",
+  "request.faraday.samson"
+].each do |metric|
+  ActiveSupport::Notifications.subscribe(metric) do |_, start, finish, _, payload|
+    duration = finish - start
+    Rails.logger.debug(message: metric, duration: duration.round(3), **payload)
+    Samson.statsd.timing metric.split(".").last(2).reverse!.join("."), duration
```

Comment:
Can we reuse or make everything in one place?
https://github.com/zendesk/samson/blob/0342c6990b7e1aa5427d982fd12661f0e901d689/config/initializers/statsd.rb#L65

Reply:
updated